pkgs/development/libraries/aspell/dictionaries.nix

```diff
@@ -153,7 +153,8 @@ let
         }
       '';
 
-      phases = [ "preBuild" "buildPhase" "installPhase" ];
+      dontUnpack = true;
+      preBuildPhases = ["preBuild"];
```

Comment:
This shouldn't be necessary unless the `buildPhase` is missing `runHook preBuild`. Can you trying building without this?

Reply:
sure but `runHook preBuild` was missing. I added it and removed the `preBuildPhases = ["preBuild"];`